src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
```

Comment:
Have you tried running with Windows Hello in IE10? I.e. I'm wondering if this line with throw an error.